packages/client/package.json

```diff
@@ -8,10 +8,29 @@
     "lint": "eslint . --ext .ts --fix"
   },
   "devDependencies": {
-    "nodemon": "^3.1.0",
-    "vitest": "^1.3.1"
+    "nodemon": "^3.1.0"
   },
   "dependencies": {
+    "@nomicfoundation/hardhat-chai-matchers": "^2.0.0",
+    "@nomicfoundation/hardhat-ethers": "^3.0.5",
+    "@nomicfoundation/hardhat-ignition": "^0.15.0",
+    "@nomicfoundation/hardhat-ignition-ethers": "^0.15.0",
+    "@nomicfoundation/hardhat-network-helpers": "^1.0.0",
+    "@nomicfoundation/hardhat-toolbox": "^5.0.0",
+    "@nomicfoundation/hardhat-verify": "^2.0.0",
+    "@nomicfoundation/ignition-core": "^0.15.0",
+    "@typechain/ethers-v6": "^0.5.0",
+    "@typechain/hardhat": "^9.0.0",
+    "@types/chai": "^4.2.0",
+    "@types/mocha": ">=9.1.0",
+    "chai": "^4.2.0",
+    "ethers": "^6.11.1",
+    "hardhat": "^2.22.2",
+    "hardhat-ethers": "^1.0.1",
+    "hardhat-gas-reporter": "^1.0.8",
+    "solidity-coverage": "^0.8.1",
+    "ts-node": "^10.9.2",
+    "typechain": "^8.3.0",
```

Comment:
are all these dependencies required?

Reply:
Almost. I will remove the ts-node and the typechain because i'm not using it anymore. But all the others ar related to hardhat configuration, tests and hardhat-tool-box.